lib/CXGN/BrAPI/v2/Results.pm

```diff
@@ -37,9 +37,8 @@ sub save_results {
     close $fh;
 
     my $search_id = substr($tempfile, -16);
-    my %result = ( searchResultDbId => $search_id );
-    #Pagination should be present in BrAPI only if data available in the result 
-    my $pagination = {}; #CXGN::BrAPI::Pagination->pagination_response(0,$page_size,$page); 
+    my %result = ( searchResultsDbId => $search_id );
```

Comment:
@MFlores2021 @darekbienkowski this change was made to bring the `result` object in line with the BrAPI 2.0 spec.  Should both keys be included in the `result` object for backwards compatibility?

Reply:
@MFlores2021 thoughts on this?